packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-reset-password/amplify-confirm-reset-password.component.html

```diff
@@ -1,63 +1,61 @@
-<div data-amplify-container>
-  <form data-amplify-form (submit)="onSubmit($event)" (input)="onInput($event)">
-    <fieldset
-      class="amplify-flex"
-      style="flex-direction: column"
-      data-amplify-fieldset
-      [disabled]="authenticator.isPending"
-    >
-      <amplify-slot name="confirm-reset-password-header" [context]="context">
-        <h3 class="amplify-heading">{{ headerText }}</h3>
-      </amplify-slot>
-      <amplify-form-field
-        name="confirmation_code"
-        type="number"
-        autocomplete="one-time-code"
-        [labelHidden]="grabField('confirmation_code', 'labelHidden', true)"
-        [placeholder]="grabField('confirmation_code', 'placeholder', null)"
-        [required]="grabField('confirmation_code', 'required', true)"
-        [label]="grabField('confirmation_code', 'label', null)"
-      ></amplify-form-field>
-      <amplify-form-field
-        name="password"
-        label="New password"
-        autocomplete="new-password"
-        [labelHidden]="grabField('password', 'labelHidden', true)"
-        [placeholder]="grabField('password', 'placeholder', null)"
-        [required]="grabField('password', 'required', true)"
-        [label]="grabField('password', 'label', 'New password')"
-      ></amplify-form-field>
-      <amplify-form-field
-        name="confirm_password"
-        type="password"
-        autocomplete="new-password"
-        [labelHidden]="grabField('confirm_password', 'labelHidden', true)"
-        [placeholder]="grabField('confirm_password', 'placeholder', null)"
-        [required]="grabField('confirm_password', 'required', true)"
-        [label]="grabField('confirm_password', 'label', 'Confirm Password')"
-      ></amplify-form-field>
+<form data-amplify-form (submit)="onSubmit($event)" (input)="onInput($event)">
```

Comment:
All I did here is remove the parent `<div data-amplify-container>`, not sure why `git` is thinking I changed the whole file :p

Reply:
Probably just because that changed the indentation?